.github/workflows/builder.yaml

```
@@ -4,9 +4,28 @@ on:
   pull_request_target:
 
 jobs:
+  check_for_membership:
+    runs-on: ubuntu-latest
+    name: Check PR author membership
+    outputs:
+      check-result: ${{ steps.composite.outputs.check-result }}
+    steps: 
+      - name: Action for membership check
+        id: composite
+        uses: hazelcast/hazelcast-tpm/membership@main
+        with:
+          organization-name: 'hazelcast'
+          member-name: ${{ github.event.pull_request.head.repo.owner.login }}
```

```suggestion
          member-name: ${{ github.actor }}
```

Otherwise I _think_ this will always fail.

---

Good catch. I was wondering about this
Tested by me making a change in the test user branch and builder passes. For sanity I made subsequent chaneg with the test user and it fails again as expect
note: does not work when `Re-run all jobs`. Both the above values point to the test user